docs/Devel/Developers.md

```diff
@@ -0,0 +1,109 @@
+
+| Script                        | Description                                                                                                                           |
+| ----------------------------- | ------------------------------------------------------------------------------------------------------------------------------------- |
+| `npm run validate-links`      | Launch script which detect links and check if they are valid. See [validateLinks.js](../../test/validateLinks.js)                     |
+| `npm run test-functional`     | Run [functional.js](../../test/functional.js).Fork [`start.js`](../../bin/start.js) and check builds and server launches              |
+| `npm run eslint`              | Run the linter. See [.eslintrc.js](../../.eslintrc.js)                                                                                |
+| `npm audit --audit-level=low` | Npm native command ([npm-audit](https://docs.npmjs.com/cli/v6/commands/npm-audit)) which check packages dependencies vulnerabilities. |
+| `npm run local-ci`            | Run CI on your local computer.                                                                                                        |
+
+
+
+### Other
+| Script                        | Description                                                                                                                           |
+| ----------------------------- | ------------------------------------------------------------------------------------------------------------------------------------- |
+| `npm run clean`               | Remove files and folders generated by `npm install` and the build scripts such as `./node_modules`, `package-lock.json`, and `./dist` |
+| `npm run clear-node`          | ***Linux Command***: Kill all node process (this is a sudo script)                                                                    |
+| `npm run reset`               | Reinstalls npm dependencies. This script runs `npm run clean` and `npm install` command                                               |
+| `npm run analyze-bundle-game` | Use [webpack-bundle-analyzer](https://www.npmjs.com/package/webpack-bundle-analyzer) to see what's inside the examples bundle         |
+| `npm run eslint-quiet`        | Run the linter without displaying warnings, only errors                                                                               |
+| `npm run eslint-fix`          | Run the linter and attempt to fix errors and warning automatically                                                                    |
+
+
+
+### Debugging with UDV library
+
+If you need to code in [Imuv](https://github.com/VCityTeam/UD-Imuv) and [UD-Viz](https://github.com/VCityTeam/UD-Viz) library you should clone the two repositories side by side on your disk. Then in the package.json of Imuv you have to link with UD-Viz library (for all fields in the package.json like `@ud-viz/*`):

```

Comment:
document also that webpack.config.resolve.modules array have to include a path to the @ud-viz node_modules folder (and be in first position in the array if I remember well)